_posts/2020-11-09-the-cloud-data-warehouse.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title: The Cloud Data Warehouse - How Do You Like Yours?
+date: 2020-11-09
+tags: cloud data warehouse snowflake bigquery redshift
+image: img/blog/sos/snowcdw/banner.jpg
+author: Paul Symons
+---
+<center><img src="/img/blog/snowcdw/banner.jpg" /></center><br/>
+
+## The Modern Analytics Engineering Stack
+
+I recently took a little time to read through [The Analytics Guide Book](https://www.holistics.io/books/setup-analytics/start-here-introduction/) by  [Holistics](https://www.holistics.io/product/). It resonates a lot with the way we think about working with data at [Mechanical Rock](https://mechanicalrock.io/), especially on the following principles:
+
+* ELT over ETL
+* SQL based analytics is the way forward
+* Use Technology To Replace Labor Whenever Possible (*So your people can focus on business value instead*)
+
+It's rare to see a book presented in such an accessible way that manages to balance complexity and insight - I recommend it to anyone looking to re-evaluate how they maximise return on investment from their data.
+
+This blog post is in two parts; in this article we will elaborate on modern Cloud Data Warehouses; in the second part we will focus on how their features allow a more flexible and agile way of delivering analytics engineering, as described in the Analytics Guide Book.
```

Comment:
Do you mean `in the first part` instead of `in this article`? Only asking because the subsequent bit says `in the second part` so it seems a bit disjointed?

Reply:
Thanks - `first part` does read better